src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -154,11 +183,24 @@ public static void login() {
     }
   }
 
+  /**
+   * Fetches a Google Application Credentials with {@code PET_SA_SCOPES} and set to {@code
+   * googleCredentials}.
```

Comment:
document that this only happens when googleCredentials is null. Maybe name it `loadAppDefaultCredentialsAsFallback()` or something.